BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -899,6 +899,18 @@ - (BOOL)isTrackingEnabled {
     return [status isEqual:kYES] ? YES : NO;
 }
 
+#pragma mark Opt out push notifications
+- (void)optOutFromPushNotifications:(BOOL)isOptedOut {
```

Comment:
I think we can follow the same naming convention for `enableTracking`. We can provide the description in the method's comments/docs about what it does under the hood.

Suggestions are, `enablePushNotifications:(BOOL)isEnabled` or `optInForPushNotifications:(BOOL)isEnabled`

Reply:
changed method names to `optInForPushNotifications ` format